cromwell-backend/src/main/scala/cromwell/backend/model/Subscription.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.backend.model
+
+/**
+  * Defines event types.
+  */
+trait EventType
```

Comment:
It will be used by different backend to publish different kind of events.
Doing this, the core does not need to be modified every time there is a need to pass something from/to backend component within Cromwell.